modules/buckets/README.md

```diff
@@ -0,0 +1,111 @@
+<!-- BEGIN MODULE HOOK -->
+# Cloud Object Storage Buckets Module
+
+This module can be used to provision and configure IBM [Cloud Object Storage](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-getting-started-cloud-object-storage) buckets.
+
+You can configure the following aspects of your instances:
+1. [Bucket encryption](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-tutorial-kp-encrypt-bucket) - based on Key Protect keys
+2. [Activity tracking](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-tracking-cos-events) and auditing
+3. [Monitoring](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-monitoring-cos)
+4. Data retention, [lifecycle](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-archive) and archiving options
+
+This module extends the root modules ability to create a single bucket, allowing multiple buckets to be defined and created with a single module call. All options available on the root module are supported. Examples are included in the basic, existing_resources and fscloud examples. The advantage of the root module, is the simplicity of creating an instance and a single bucket, it offer ease of use for small deployments. The advantage of the buckets module is the ability to create all required buckets in a single place. With scale comes complexity and the ability to keep the entire definition in one place helps retain clarity on the bucket properties. The outputs for all the buckets are included in a single map, again improving clarity and enabling the results to be iterated over.
```

Comment:
`root module's ability...`

but I prefer to rewrite without anthropomorphizing the module (it really has no abilities) and possession, as in

> The submodule extends the root module by including support for multiple buckets....